### ISSUE TITLE
Feedback about DROP

### ISSUE THREAD

**Issue by D3vl0per** [D3vl0per/Valorant-watcher#8]
Please send response about DROP if you get one.

**Comment by D3vl0per** [comment]
Thanks for support and response [Superman](https://www.buymeacoffee.com/D3v/c/184614).

**Comment by alanferreiravalorant** [comment]
First: Thank you for this release its very cool

Tutorial for windows

**-You need to download [chromium](https://download-chromium.appspot.com/) (extract in your computer)

-Open cmd and navigate to folder bot like a this: cd C:\Users\you-username\Downloads\bot
-type npm install
-type npm start
-put your auth-key
-put commanline for exe chromium like a this:
C:\Users\you-username\Downloads\chrome\chrome.exe
Press enter and done.

FIXES:
**If you get error restart the bot (CTRL+C) and "npm start" again in cmd.

**Comment by Relyze** [comment]
I got a drop a couple days ago after running this bot for  36 hours, thanks chief, got 4 drops on 4 accounts all around the same time using this.
got the drop on windows btw, I used powershell, and just did this:
npm install
npm start
key, and expiration
Profit?


note: npm install auto installed chromium to allow for this to run, so you don't need chromium yourself, npm install will install it by itself, at least in my case.

**Comment by LevitatingBusinessMan** [comment]
@Relyze This package uses puppeteer-core, so npm should not install chromium by itself

**Comment by D3vl0per** [comment]
That's true!
I think you use older release @Relyze.

**Comment by LevitatingBusinessMan** [comment]
@D3vl0per This sure seems over engineered lmao, I made a similiar tool https://github.com/LevitatingBusinessMan/puppeteer-twitch-login but it actually fills in the credentials into the login window key for key. I hoped this would reliably log me in without Twitch suspecting me to be a bot but eventually it started asking for an anti bot test and it pretty much always needed email verification. Your implementation should be a lot more stable. Anyway I got my key so I am done with this, I assume you are farming keys to sell?

**Comment by LevitatingBusinessMan** [comment]
Oh and btw, @D3vl0per I am 90% certain puppeteer works just as well with Google Chrome, which most people (on windows) have installed already.

**Comment by Relyze** [comment]
You're right, actually I completely forgot, I used npm install chromium, I just haven't had to use the tool in a while, but now that you mention it, you're right, sorry for the false information lmfao.

EDIT: nvm, I legit just use a few day old release where it was installed on it's own, I just checked. my bad

**Comment by D3vl0per** [comment]
I will start a gitter, because this issue purpose just for successful DROP!
https://gitter.im/valorant-watcher/community#

**Comment by michaelmessi** [comment]
I got drop key since 2h thank you for nice bot <# :"D

**Comment by rgvxlens** [comment]
Working as intended. Just need to put "chrome.exe" eg " C:\Users\you-username\Downloads\chrome\chrome.exe" when asked for Chrome's path.

**Comment by paname75** [comment]
Got a drop too after about 24h of the docker app running ! thanks.

**Comment by Xyz3R** [comment]
Worked decent for me. I got 2 drops on 2 account. First one after around 18h, second one after 36h. Was running the tool in a docker stack 2x for 2 accs.

**Comment by KiwisKiwi** [comment]
Got a drop after 2 days with default settings of this bot.

**Comment by MiguelAngel2345** [comment]
Me too i finally got it

**Comment by D3vl0per** [comment]
Thanks for donation :smile: : https://www.buymeacoffee.com/D3v/c/192808
Köszi a támogatást lako!

**Comment by QuentinGruber** [comment]
Got a drop after 20h of waiting, thx a lot for the bot ❤️

**Comment by D3vl0per** [comment]
@QuentinGruber Thanks for donation!

**Comment by rvkolosov** [comment]
Hi, I launched 10 watchers on the server 2 hour ago, does your image use a proxy?
Thanks for the package!)

**Comment by D3vl0per** [comment]
> Hi, I launched 10 watchers on the server 2 hour ago, does your image use a proxy?
> 
> Thanks for the package!)

Yes, you can set proxy if you want! 
Please read the README @rvkolosov or start a new issue! Thanks!

**Comment by almoghamdani** [comment]
Got 2 drops by using 2 docker containers (both in about 36 hours). Thanks man :)

**Comment by rvkolosov** [comment]
> > Hi, I launched 10 watchers on the server 2 hour ago, does your image use a proxy?
> > Thanks for the package!)
> 
> Yes, you can set proxy if you want!

Do you have instructions for this?

**Comment by jvogit** [comment]
1 drop in about 13 hours. Thanks!

**Comment by corediggr** [comment]
yes, got a drop after around 8hrs of using ur bot. but i also watched a lot the last couple of days...

**Comment by rvkolosov** [comment]
Awesome!
9/10, after 30-36 hours

**Comment by D3vl0per** [comment]
Thanks for donation: https://www.buymeacoffee.com/D3v/c/195817

**Comment by theunrealmachine** [comment]
Got drop, but only after ~40 hours. Thank you!

**Comment by D3vl0per** [comment]
@rvkolosov Thanks for donation: https://www.buymeacoffee.com/D3v/c/196766

**Comment by OMKE** [comment]
Got the key, thanks man

**Comment by adagari** [comment]
git clone and docker build this morning, had a key 11 hrs later. Thanks.

**Comment by giovanni-orciuolo** [comment]
Just want to say that me and my friend got the key thanks to this program after 2 days of farming! Thank you!!!

**Comment by TheExmax223** [comment]
Got key after 30hrs, thank you!

**Comment by ghost** [comment]
Got key after not even a day. Thank you (run it on a VPS for more chances)

**Comment by dimorinny** [comment]
Got it after about 20 hours. Thank you

**Comment by noaahh** [comment]
Got three drops with three docker containers running parallel in one week! Thanks

**Comment by SeasonalFerret** [comment]
Got a drop, thank you!

Ran program for 13 hours, left program off overnight, got it this morning without opening the program today.

**Comment by 0xAhmet** [comment]
I got 8 drops total. Thank you.

**Comment by D3vl0per** [comment]
Thanks for supporting: https://www.buymeacoffee.com/D3v/c/209734

**Comment by afeuerstein** [comment]
> 
> 
> Got three drops with three docker containers running parallel in one week! Thanks

Did you use a proxy? @noaahh

**Comment by noaahh** [comment]
> > Got three drops with three docker containers running parallel in one week! Thanks
> 
> Did you use a proxy? @noaahh

No, I didn't.

**Comment by clementDele** [comment]
Got my key in about 40 hours :) thank you 👍

**Comment by ghost** [comment]
![image](https://user-images.githubusercontent.com/58285036/81028801-de84e280-8e50-11ea-8ef8-49ca4f5b28fe.png)
Took 106 hours to get it. (my luck knows no bounds)

**Comment by spheppner** [comment]
Got my drop after about 1 week or more. Even though it took pretty long, I really really thank you. Nice and clean coding! :)

**Comment by giacomoferretti** [comment]
Got 8 drops for me and my friends. Thank you very much!

_NOTE: you have to wait at least 48h to get the email.
Ran using Docker 24/7 on ARM._

**Comment by thvozdovic** [comment]
20-30 hours. thank you so much

**Comment by afeuerstein** [comment]
Got a drop after ~30-35h. Thanks for your support!

**Comment by D3vl0per** [comment]
Thanks for @Ziyad32 to this video: https://youtu.be/bwzv7wT44Ds :smile:

**Comment by Ziyad32** [comment]
Drop after 5 hours (i'm lucky)

**Comment by dorianlaf05** [comment]
Got a drop after less than 24h, thank you so much!

**Comment by Sasa993** [comment]
Got a key after 20-25 hours of running the script on Linux. Sending you a couple of "thank you" bucks in a few days.
Thank you!

**Comment by afeuerstein** [comment]
Got my 2nd drop after 20h

**Comment by chriscoffee** [comment]
Invited after ~30h running on k8s. Much appreciated :+1:

**Comment by denizoral** [comment]
Got my key after 25 hours on raspbbery pi! Thank you!

**Comment by King-BR** [comment]
Got dropped after ~40 hours
